src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/util/SalesforceQueryExecutionException.java

```diff
@@ -0,0 +1,39 @@
+package io.cdap.plugin.salesforce.plugin.source.batch.util;
```

Comment:
this statement should be below the license.

Reply:
isRetryRequired = Boolean.valueOf(conf.get(SalesforceSourceConstants.CONFIG_RETRY_REQUIRED, String.valueOf(true)));
 this change has already been done in  SalesforceBulkRecordReader in initialize

Reply:
Done